src/spec-node/devContainers.ts

```diff
@@ -133,6 +138,11 @@ export async function createDockerParams(options: ProvisionOptions, disposables:
 		skipPostAttach: options.skipPostAttach,
 		experimentalImageMetadata: options.experimentalImageMetadata,
 		skipPersistingCustomizationsFromFeatures: options.skipPersistingCustomizationsFromFeatures,
+		dotfilesConfiguration: {
+			repository: options.dotfiles.repository,
+			installCommand: options.dotfiles.installCommand,
+			targetPath: options.dotfiles.targetPath || '~/dotfiles',
```

Comment:
If you apply the suggestion below (which adds the default property ), then can avoid this.

```suggestion
			targetPath: options.dotfiles.targetPath,
```